### PR TITLE
Mover type for TFinity movers was not defined.

### PR DESCRIPTION
**Pull request by moibenko**
This caused incorrect insert into drive_utilization and problems with utulization plots.

### CODE REVIEW

File: src/ratekeeper.py [hunk 1]
```diff
@@ -357,18 +371,25 @@ def update_DRVBusy(self, mcc):
...
+                Trace.trace(10, 'CALLING inquire_vol %s' % (drive['volume'],))
                 v_info=self.vcc.inquire_vol(drive['volume'])
-                sg=None
-                try:
-                    sg=volume_family.extract_storage_group(v_info['volume_family'])
-                except:
-                    exc, msg, tb = sys.exc_info()
+                Trace.trace(10, 'inquire_vol returned %s' % (v_info,))
+                time.sleep(1)
+                sg='UNKNOWN'
```
Comment (DmitryLitvintsev) [inline src/ratekeeper.py:379]: why is this sleep needed?

File: src/ratekeeper.py [hunk 2]
```diff
@@ -357,18 +371,25 @@ def update_DRVBusy(self, mcc):
                 #busy counts cant have it yet.
 
             if drive['volume']:
+                Trace.trace(10, 'CALLING inquire_vol %s' % (drive['volume'],))
```
Comment (DmitryLitvintsev) [inline src/ratekeeper.py:375]: I thought you wanted to get rid of calling inquire_vol?

File: src/ratekeeper.py [hunk 3]
```diff
@@ -341,13 +344,24 @@ def update_DRVBusy(self, mcc):
                         m_status = movc.status(self.mover_to, self.mover_retries)
                         if e_errors.is_ok(m_status):
                             valid_drives.append(m_status['media_changer_device'])
-
+                            # find in drives list
+                            for drive in drives_list:
+                                if drive['name'] == m_status['media_changer_device']:
+                                    if not drive['type']:
+                                        #Trace.trace(10, 'UPDATING %s' % (drive,))
```
Comment (DmitryLitvintsev) [inline src/ratekeeper.py]: remove commented line

File: src/ratekeeper.py [hunk 4]
```diff
@@ -341,13 +344,24 @@ def update_DRVBusy(self, mcc):
                         m_status = movc.status(self.mover_to, self.mover_retries)
                         if e_errors.is_ok(m_status):
                             valid_drives.append(m_status['media_changer_device'])
-
+                            # find in drives list
+                            for drive in drives_list:
+                                if drive['name'] == m_status['media_changer_device']:
+                                    if not drive['type']:
```
Comment (DmitryLitvintsev) [inline src/ratekeeper.py]: suggest to leave a comment why drive['type'] can be None and why the lookup needs to be done. 
As I understand lack of drive type when querying TFF1 broke the plotting of active drives.

File: src/ratekeeper.py (hunk 1 above)
Comment (moibenko) [reply src/ratekeeper.py:379]: I put it there to reduce the request rates

File: src/ratekeeper.py (hunk 2 above)
Comment (moibenko) [reply src/ratekeeper.py:375]: I did, but decided to leave as I see it working, although now always good.

File: src/ratekeeper.py (hunk 4 above)
Comment (moibenko) [reply src/ratekeeper.py]: I do not know why. This was done according to your comments that the drive type was ''. My impression was that it affects plots. So I do not know what comment to put here.

File: src/ratekeeper.py (hunk 3 above)
Comment (moibenko) [reply src/ratekeeper.py]: removed

File: src/ratekeeper.py (hunk 4 above)
Comment (DmitryLitvintsev) [reply src/ratekeeper.py]: I think you misunderstood my comment.  I wanted o have this 

```
 if not drive['type']:
....
```

bit commented. "That TFF1 does not have a type defined and this is how we get that type in in this case" or something along these lines if above is not correctly describing  why drive['type'] == None

**Review by DmitryLitvintsev — APPROVED** [review]

**Pull request merged**